ReadEtextsActivity.py

```diff
@@ -586,13 +589,13 @@ def reset_play_button(self):
         self.textview.grab_focus()
 
     def highlight_next_word(self, word_count):
-        print 'word_count', word_count
         if word_count < len(self.word_tuples):
             word_tuple = self.word_tuples[word_count]
             textbuffer = self.textview.get_buffer()
             iterStart = textbuffer.get_iter_at_offset(word_tuple[0])
             iterEnd = textbuffer.get_iter_at_offset(word_tuple[1])
             bounds = textbuffer.get_bounds()
+            textbuffer.remove_all_tags(bounds[0], bounds[1])
             textbuffer.apply_tag(self.normal_tag, bounds[0], iterStart)
```

Comment:
@jdsimmons I'd like to know why you applied the self.normal_tag here as it doesn't have any effect on the text highlight.